src/CachePool.php

```diff
@@ -40,13 +40,18 @@ class CachePool implements CacheInterface
      * @var mixed
      */
     protected $defaultValue;
+    /**
+     * @var int|DateInterval
+     */
+    protected $defaultTtl;
 
     /**
      * @param wpdb   $wpdb         The WP database object.
      * @param string $poolName     The name of this cache pool. Must be unique to this instance.
      * @param mixed $defaultValue  A random value. Used for false-negative detection. The more chaotic - the better.
+     * @param int|DateInterval $defaultTtl Default TTL to use when caching new entries.
```

Comment:
Good thing I do not enforce a PHPCS standard. Don't really care that much if the docs are aligned. And trust you to do it right enough. :)

Reply:
I very much care if their aligned or not. It took everything in me to not hit Ctrl+Shift+F in PHPStorm. But I saw that the existing ones weren't formatted, so I tried to keep everything the same.